src/common/constants.js

```diff
@@ -0,0 +1,32 @@
+const LOG_MESSAGE = Object.freeze({
+  START_MESSAGE: "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분",
+  ATTEMPT_COUNT_MESSAGE: "시도할 횟수는 몇 회인가요?",
+  WINNERS_ANNOUNCEMENT: '최종 우승자 : ',
+})
+
+const ERROR_MESSAGE = Object.freeze({
+  INVALID_INPUT: "자동차 이름은 최대 5글자까지 가능합니다. 다시 입력해 주세요.",
```

Comment:
INVALID_INPUT 보다는 더 직관적인 에러 메시지가 좋을 것 같아요 ! (다른 변수명이랑 맞추려다 보니 아래와 같이 적긴 했지만, 쨌든 포인트는 조금 더 직관적인 변수명이 좋을 것 같아서요!)
```suggestion
  TOO_LONG_INPUT: "자동차 이름은 최대 5글자까지 가능합니다. 다시 입력해 주세요.",
```


(에러 메시지들 끝에 있는 느낌표 귀엽네요 ☺️)

Reply:
다른분들이 보기에 훨씬 더 좋은 변수명인것 같습니다! 감사합니다 ㅎㅎ

Reply:
저는 최근에 EXCEED 라는 표현이 눈에 들어오더라구요! 
`EXCEEDED_MAX_LENGTH`
`EXCEED_MAX_LENGTH`
만약 여기에 적용하면 이런 느낌이 될 것 같아요!~